src/System.Net.Security/src/System/Net/_SSPIWrapper.cs

```diff
@@ -5,229 +5,43 @@
 using System.Globalization;
 using System.Net.Security;
 using System.Runtime.InteropServices;
-
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
 namespace System.Net
 {
     internal static class SSPIWrapper
```

Comment:
Design: (I think this is against what I've originally proposed... sorry.)
We need to get code back in here and push this in PAL. I need this for NegotiateStream, NTAuthentication, etc.